packages/react/src/components/Storage/File-Uploader/FileUploader.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Can we remove the unused variables instead of turning off linting

Reply:
Ok